packages/dds/sequence/src/intervalCollection.ts

```diff
@@ -1956,6 +1956,13 @@ export class IntervalCollection<TInterval extends ISerializableInterval> extends
 		}
 	}
 
+	/**
+	 * @deprecated - This functionality was useful when adding two intervals at the same start/end positions resulted
+	 * in a conflict. This is no longer the case (as of PR#6407), as interval collections support multiple intervals
+	 * at the same location and gives each interval a unique id.
+	 *
+	 * As such, the conflict resolver is never invoked and unnecessary. This API will be removed in an upcoming release.
+	 */
 	public addConflictResolver(conflictResolver: IntervalConflictResolver<TInterval>): void {
```

Comment:
can you add a note to the breaking.md upcoming section, that's where we notify about deprecations that will be removed

Reply:
yeah good call. i've been doing it in next mostly with the removal (most breaks I've made are straightforward fixes) but makes more sense to do it with the deprecation. i'll do that here and in the future.